airbyte-webapp/src/packages/cloud/views/auth/LoginPage/LoginPage.tsx

```diff
@@ -100,9 +100,9 @@ const LoginPage: React.FC = () => {
                 >
                   <FormattedMessage id="login.forgotPassword" />
                 </Link>
-                <LoadingButton className={styles.logInBtn} type="submit" isLoading={isSubmitting}>
+                <Button customStyles={styles.logInBtn} type="submit" isLoading={isSubmitting}>
```

Comment:
Nit: I think className was the right call here, but probably not worth doubling back.

Reply:
+1

Reply:
I was trying to use className, but had some issues with className usage since there was some SCSS overriding there, so decided to use another porperty

Reply:
Could we dig into why this happened with className? We should use the standard name for this, to make our API clear and not requiring consumers to learn custom properties on every component.

Reply:
ok, I will try to investigate on this

Reply:
find out. the issue was passing every property from the top into the native button HTML element. fixed that